shanghai/2020/README.md

```diff
@@ -0,0 +1,2 @@
+# wikix
+Hello, I'm wikix from China
```

Comment:
welcome! could you please rename this file to another name that's not `README.md`? otherwise LGTM.

Reply:
Got it, I will rename it.